tools/docker/Dockerfile.aspire.assistant

```diff
@@ -0,0 +1,57 @@
+# syntax=docker/dockerfile:1
+
+# Dockerfile for assistants
+# Context root is expected to be the root of the repo
+ARG python_image=python:3.11-slim
+
+# These build arguments will differ per assistant:
+# package is the directory name of the assistant package under /assistants
+ARG package
+ARG app
+
+FROM ${python_image} AS build
+
+ARG package
+
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /bin/uv
+
+# copy all library packages
+COPY ./libraries/python /packages/libraries/python
+# copy the assistant package
+COPY ./assistants/${package} /packages/assistants/assistant
+
+# install the assistant and dependencies to /.venv
+RUN uv sync --directory /packages/assistants/assistant --no-editable --no-dev --locked
+
+FROM ${python_image}
+
+ARG app
+
+# BEGIN: enable ssh in azure web app - comment out if not needed
+########
+# install sshd and set password for root
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    openssh-server \
+    && rm -rf /var/lib/apt/lists/* \
+    && echo "root:Docker!" | chpasswd
+
+# azure sshd config
+# COPY ./tools/docker/azure_website_sshd.conf /etc/ssh/sshd_config
+# ENV SSHD_PORT=2222
```

Comment:
I needed to remove this to deploy to ACA with Aspire, but if I remove those lines from the prior existing Dockerfiles, the CI piplines will break and current deployment won't work. I opted to duplicate this rather than having a big impact on current project status